admin-ui/plugins/admin/redux/sagas/ApiPermissionSaga.js

```diff
@@ -121,7 +122,17 @@ export function* deletePermission({ payload }) {
     })
     return data
   } catch (e) {
-    yield put(updateToast(true, 'error'))
+    const actionData = payload?.action?.action_data
+    const apiPermissions = yield select((state) => state.apiPermissionReducer.items || [])
+    const rolePermissionMapping = yield select((state) => state.mappingReducer.items || [])
+    const finalMessage = buildPermissionDeleteErrorMessage(
+      e,
+      actionData,
+      apiPermissions,
+      rolePermissionMapping,
+    )
```

Comment:
Verify that the error parameter is utilized in the utility function.

The `buildPermissionDeleteErrorMessage` receives the error object but the relevant code snippet shows it only uses `actionData`, `apiPermissions`, and `rolePermissionMapping` in its body. If the error isn't used, consider removing it from the signature or document why it's reserved for future use.